src/MongoDB.Driver.Core/Core/Operations/EndTransactionOperation.cs

```diff
@@ -112,7 +127,8 @@ protected virtual BsonDocument CreateCommand()
             return new BsonDocument
             {
                 { CommandName, 1 },
-                { "writeConcern", () => _writeConcern.ToBsonDocument(), !_writeConcern.IsServerDefault }
+                { "writeConcern", () => _writeConcern.ToBsonDocument(), !_writeConcern.IsServerDefault },
+                { "recoveryToken", _recoveryToken, _recoveryToken != null }
```

Comment:
There is no longer any reason for CreateDocument to be virtual, nothing overrides it.

Why don't we make it private again like it was originally?

Reply:
Good idea: It's private again!